L/Lua/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+using BinaryBuilder
+
+name = "Lua"
+version = v"5.3.5"
+
+sources = [
+    "https://www.lua.org/ftp/lua-5.3.5.tar.gz" =>
+        "0c2eed3f960446e1a3e4b9a1ca2f3ff893b6ce41942cf54d5dd59ab4b3b058ac",
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/lua-*
+
+if [[ $target == *-apple-* ]]; then
+    MAKE_TARGET=macosx
+elif [[ $target == *-freebsd* ]]; then
+    # NOTE: The FreeBSD ports build also uses the bsd instead of the freebsd target,
+    # since it hard-codes fewer things, apparently
+    MAKE_TARGET=bsd
+elif [[ $target == *-mingw* ]]; then
+    MAKE_TARGET=mingw
+else
+    MAKE_TARGET=linux
+fi
+
+make -j${nproc} $MAKE_TARGET
+make install
+"""
+
+platforms = supported_platforms()
+
+products = [
+    ExecutableProduct("lua", :lua),
+    LibraryProduct("liblua", :liblua),
+]
+
+dependencies = []
```

Comment:
Homebrew sets readline as a dependency and FreeBSD Ports uses libedit, apparently optionally. Assuming it's optional, we shouldn't need a dependency at all, I would think...

Reply:
[Arch Linux](https://www.archlinux.org/packages/extra/x86_64/lua/) and [Debian](https://packages.debian.org/stretch/lua5.3) have readline has dependency.  The good point is that we should have everything to build readline, that is just ncurses

Reply:
It appears to be hard-coded as a dependency for Linux, but not for the others.